tg/views/__init__.py

```diff
@@ -22,6 +27,24 @@
 )
 
 
+class ViewComponent(metaclass=ABCMeta):
+    win: window
+    refresh_stoped: bool = False
+
+    @abstractmethod
+    def resize(self, p: float) -> None:
+        pass
+
+    @abstractmethod
+    def draw(self, *args, **kwargs) -> None:
+        pass
+
+    def refresh(self) -> None:
+        if self.refresh_stoped:
```

Comment:
I am on purpose made an implementation which does not requires to check for bool variable on every refresh. Don't you think it's better method? I mean changing implementation instead of checking condition?